$NetBSD$

Not for sure the difference webkit-gtk and webkit2gtk


--- CMakeLists.txt.orig	2025-01-31 22:47:20.000000000 +0900
+++ CMakeLists.txt	2025-02-22 18:23:37.718589221 +0900
@@ -82,7 +82,7 @@ set(libpst_minimum_version 0.6.54)
 set(libsecret_minimum_version 0.5)
 set(libxml_minimum_version 2.7.3)
 set(shared_mime_info_minimum_version 0.22)
-set(webkit2gtk_minimum_version 2.34.0)
+set(webkit-gtk_minimum_version 2.34.0)
 set(sqlite_minimum_version 3.7.17)
 
 # Optional Packages
